### ISSUE TITLE
3.2.0 Wizard Live-Z "Try Again" bug

### ISSUE THREAD

**Issue by phooton** [prusa3d/Prusa-Firmware#732]
Run Wizard on 3.2.0 final and when I repeat Live-Z test the extruder always seem to start from same height, but the shown Z offset stays at the value from previous run. Making the Z offset calibration unusable unless I hit it the very first time correctly?

**Comment by stahlfabrik** [comment]
I guess yes. 

Do it only once. Then continue adjusting live z with a normal print. The Prusa Logo is fine for that for example

**Comment by alhazar313** [comment]
Encountered the same issue

**Comment by Panayiotis-git** [comment]
Same here, with MK2.5 v3.2.0. First run was -650, second -1320, third run was -2050.

**Comment by layneok** [comment]
Same bug found here, ended up putting huge scratch and dent on spring steel sheet. Bummer!

![img_1712](https://user-images.githubusercontent.com/9872743/39937669-51d87f26-5505-11e8-9438-22d610c8a49f.jpg)

**Comment by Zaf9670** [comment]
Can you all provide which printer you are using as well? Just wondering if this is isolated to the Mk2/Mk2.5 or if it also affects the Mk3.

**Comment by alhazar313** [comment]
Mine is a MK3 and has the issue

**Comment by layneok** [comment]
Mine is also a MK3.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by josepvm** [comment]
Same issue for me in a Mk3, after upgrading to 3.2.0 final.

**Comment by PavelSindler** [comment]
Thank you for reporting this bug. PR https://github.com/prusa3d/Prusa-Firmware/pull/747 should fix this.

**Comment by ram64** [comment]
I have the same issue with a MK2.5 upgrade. When using the Wizard to calibrate the printer I've been asked if I want to retry the Live Z adjust and I said "Yes". The operation started from the beginning with the nozzle height unadjusted, but the value remained from the first try. I had to double the value effectively to get the results. On the first print the nozzle dragged across the bed scraping it. I've had to reset XYZ calibration and start over.
